static/bidder-info/algorix.yaml

```diff
@@ -8,3 +8,8 @@ capabilities:
       - banner
       - video
       - native
+  site:
```

Comment:
Please create exemplary JSON test cases for each of these. I suggest copying the following files, renaming and tweaking accordingly:
`adapters/algorix/algorixtest/exemplary/sample-banner.json`
`adapters/algorix/algorixtest/exemplary/sample-video.json`
`adapters/algorix/algorixtest/exemplary/sample-native.json`

Reply:
done